src/seedu/addressbook/data/person/Address.java

```diff
@@ -1,59 +0,0 @@
-package seedu.addressbook.data.person;
```

Comment:
As point out in the tutorial, you should keep the `Address` class. And the `Address` class contains `Block` `Street` `Unit` `PostalCode` (Composition, you will learn it later)